buildSrc/src/main/java/Dependencies.kt

```diff
@@ -21,14 +22,10 @@ object Libs {
     val ASM = "org.ow2.asm:asm-util:${LibsVersion.ASM}"
     val SQLITE = "androidx.sqlite:sqlite:${LibsVersion.SQLITE}"
     val SQLITE_FRAMEWORK = "androidx.sqlite:sqlite-framework:${LibsVersion.SQLITE}"
+    val SENTRY_ANDROID = "io.sentry:sentry-android:${LibsVersion.SENTRY}"
```

Comment:
why moving from Samples to Libs? the SDK is still used only by the samples and/or tests

Reply:
I thought it would look weird to use Samples.Room.Sentry dependency in the plugin-build (which is an included build, and in theory should be agnostic of other composite build's projects)

Reply:
makes sense if we need in the plugin-build, maybe we have to decide about this first -> https://github.com/getsentry/sentry-android-gradle-plugin/pull/190#discussion_r721611650